azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -66,6 +68,8 @@ def ingest_from_file(self, file_descriptor: Union[FileDescriptor, str], ingestio
         :param azure.kusto.ingest.IngestionProperties ingestion_properties: Ingestion properties.
         """
         file_descriptor = FileDescriptor.get_instance(file_descriptor)
+        ingestion_properties.application_for_tracing = self.application_for_tracing
```

Comment:
Code duplication between the two methods

Reply:
shouldnt i set this no matter which option the user chose? and i cant just put it at the end in ingest_from_blob once because the ingestion properties are handled beforehand (and i dont know in what phase the DM will need it)